### PR TITLE
Adding endpoint for status in batch actions 

### PR DESCRIPTION
**Pull request by wiewiurdp**
# Description
Adding endpoint for status in batch actions 

Issue # 
#1283 

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [ ] Unit Tests
- [x] Behat Test

# Checklist:
- [x] I have read the contribution requirements and fulfill them.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by rprzedzik — APPROVED** [review]

File: module/batch-action/src/Infrastructure/Persistence/Query/DbalBatchActionQuery.php [hunk 1]
```diff
@@ -65,6 +67,44 @@ public function getInformation(BatchActionId $id, Language $language): BatchActi
...
+                                 where batch_action_id = ba.id
+                                   and success = true
+                                   and fail_reason is null) as succeeded')
+            ->addSelect('(select count(*)
+                                 from batch_action_entry
+                                 where batch_action_id = ba.id
+                                   and fail_reason is not null) as errors')
+            ->orderBy('started_at', 'DESC')
+            ->from(self::TABLE_BATCH_ACTION, 'ba')
+            ->setMaxResults(10)
```
Comment (wfajczyk) [inline module/batch-action/src/Infrastructure/Persistence/Query/DbalBatchActionQuery.php:103]:
```suggestion
            ->setMaxResults(self::PROFILE_RESOULT)
```
Magick number to const

**Review by wfajczyk — CHANGES_REQUESTED** [review]

**Review by wfajczyk — APPROVED** [review]

**Review by rprzedzik — DISMISSED** [review]

File: module/batch-action/features/batch-action-notification.feature [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Feature: batch action module profile notification
+
+  Background:
+    Given I am Authenticated as "test@ergonode.com"
+    And I add "Content-Type" header equal to "application/json"
+    And I add "Accept" header equal to "application/json"
+
+  Scenario: Get export info
```
Comment (piotrkreft) [inline module/batch-action/features/batch-action-notification.feature:8]:
```suggestion
  Scenario: Get batch action info
```

**Review by piotrkreft — CHANGES_REQUESTED** [review]

File: module/batch-action/src/Application/Controller/Api/Notification/BatchActionNotificationAction.php [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+class BatchActionNotificationAction
+{
+    private BatchActionQueryInterface $query;
+
+    public function __construct(BatchActionQueryInterface $query)
+    {
+        $this->query = $query;
+    }
+    /**
+     * @IsGranted("CHANNEL_GET_NOTIFICATION")
```
Comment (wfajczyk) [inline module/batch-action/src/Application/Controller/Api/Notification/BatchActionNotificationAction.php:31]: Channel ?
```
     * @IsGranted("CHANNEL_GET_NOTIFICATION")
```

File: module/batch-action/src/Resources/config/routes.yml [hunk 4]
```diff
@@ -2,3 +2,8 @@ ergonode_batch_action_api:
     resource: '../../Application/Controller/Api/*'
     type: 'annotation'
     prefix: '/api/v1/{language}'
+
+ergonode_channel_notification_api:
```
Comment (wfajczyk) [inline module/batch-action/src/Resources/config/routes.yml:6]:
```suggestion
ergonode_batch_action_notification_api:
```

**Review by wfajczyk — CHANGES_REQUESTED** [review]

**Review by piotrkreft — APPROVED** [review]

**Review by wfajczyk — APPROVED** [review]

**Review by rprzedzik — APPROVED** [review]

**Pull request merged**